src/schema.cpp

```diff
@@ -88,6 +88,19 @@ void Schema::validate() const
     }
 }
 
+namespace {
+struct IsNotRemoveProperty {
+    bool operator()(SchemaChange sc) const { return sc.visit(*this); }
```

Comment:
Ok. I only ask because I see it being taken by reference in other places when used as an argument.